test/simulator/timestamp/src/session_simulator.cpp

```diff
@@ -456,6 +474,12 @@ session_simulator::is_read_ts_set() const
     return (_read_ts != 0);
 }
 
+bool
+session_simulator::is_txn_error() const
```

Comment:
If this function is not used, I don't think we should have it. This is new and untested code.

Reply:
Agreed